js/actions/test.js

```diff
@@ -26,7 +26,9 @@
 
 const Parse = require('parse/react-native');
 const ActionSheetIOS = require('ActionSheetIOS');
+const Platform = require('Platform');
 const {version} = require('../env');
+const SendIntentAndroid = Platform.OS === 'android' ? require('react-native-send-intent') : undefined;
```

Comment:
FYI the `require`s are resolved at build time, i.e. it doesn't matter if you do `Platform.OS === 'android'` – these will get included into the bundle anyways. Looking at [that module's source](https://github.com/lucasferreira/react-native-send-intent/blob/master/index.js) it seems it's totally safe to require this module on iOS anyways, as long as we are not calling methods on it.